pcsx2/GS/Renderers/Common/GSRenderer.cpp

```diff
@@ -300,7 +300,11 @@ static GSVector4 CalculateDrawDstRect(s32 window_width, s32 window_height, const
 		targetAr = 4.0f / 3.0f;
 	}
 	else if (EmuConfig.CurrentAspectRatio == AspectRatioType::R16_9)
+	{
 		targetAr = 16.0f / 9.0f;
+	}
+	else if (EmuConfig.CurrentAspectRatio == AspectRatioType::R10_7)
+		targetAr = 10.0f / 7.0f;
```

Comment:
Please make single statement if/else blocks consistent. Generally we prefer to use curly braces for them.